packages/blocks/README.md

```diff
@@ -696,6 +694,29 @@ _Parameters_
 
 Registers a new block style variation for the given block.
 
+_Usage_
+
+```js
+import { __ } from '@wordpress/i18n';
+import { registerBlockStyle } from '@wordpress/blocks';
+import { Button } from '@wordpress/components';
+
+const ExampleComponent = () => {
+	return (
+		<Button
+			onClick={ () => {
+				registerBlockStyle( 'core/quote', {
+					name: 'fancy-quote',
+					label: __( 'Fancy Quote' ),
```

Comment:
One thing that should be included as a note is how CSS styles can be wired with a newly registered variation. Maybe it's enough to link to the documentation that explains all the details:

https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-styles.md#styles